### PR TITLE
Hpc service

### PR DESCRIPTION
**Pull request by skavhaug**
This PR adds some text to the HPC section of the services page. Please feel free to iterate wildly!

### CODE REVIEW

**Comment by skavhaug** [comment]
@eigilhs Perhaps you can apply your magic touch when it comes to making stuff look great? :)

**Comment by eigilhs** [comment]
> @eigilhs Perhaps you can apply your magic touch when it comes to making stuff look great? :)

:smile: Will take a look tonight.

**Comment by alexanfl** [comment]
@eigilhs mr css

File: content/services/hpc/index.md [hunk 1]
```diff
@@ -7,4 +7,28 @@ resources:
...
 
-HPC is about getting computers to do their shit in a hurry.
+Getting maximum compute power from limited hardware resources has always been a
+main challenge in compute intensive applications, be it numerical solution of
+partial differential equations or Monte Carlo integration to compute the  
+multidimensional definite integral of some interesting function. 
+
+Ever since the beggining of the computer era, technologist, programmers and
+scientists have carefully written instructions to the CPU to solve ever more
+challenging problems spending less time and energy. HPC is traditionally the
```
Comment (mandus) [inline content/services/hpc/index.md]: Even in this context, introduce the abbreviation by stating `High Performance Computing (HPC)` the first time it is seen

Comment (skavhaug) [reply content/services/hpc/index.md]: Fair point :)

**Comment by skavhaug** [comment]
All contributions from @expertanalytics/core are extremely welcome. Let's make `xal.no` great again!

**Pull request merged**